<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinData</class>
 <widget class="QWidget" name="WinData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QStackedWidget" name="WidgetShow">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="WidgetTestData">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="TabTestData">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WidgetDetail">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <item>
        <widget class="QTableWidget" name="TabDetail">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>项目</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>测试总数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>合格数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>不合格数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>合格率</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="WinDetail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>合格率</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="Histogram" name="TextHistogram">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="WidgetCtrl" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>型号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="DateEdit1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>至</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="DateEdit2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2017</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BtnShow">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>全部导出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCheck">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnClear">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnExit">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Histogram</class>
   <extends>QLabel</extends>
   <header>Histogram.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
